docs/content/en/docs/topics/restore-a-backup/_index.md

```diff
@@ -66,13 +80,49 @@ kubectl apply -f my-medusa-bucket-key.yaml
+
+```
+size: 3
+backupRestore: 
+  medusa:
+    enabled: true
+    bucketName: k8ssandra-bucket-dev
+    bucketSecret: medusa-bucket-key
+    storage: s3
+```
+
+The chart's entries relate to a Kubernetes Secret, which contains the object store credentials. Specifically, the `bucketSecret` property specifies the name of a secret that should contain an AWS access key. As described in the [Medusa documentation](https://github.com/thelastpickle/cassandra-medusa/blob/master/docs/aws_s3_setup.md), the AWS account with which the key is associated should have the permissions that are required for Medusa to access the S3 bucket.
+
+Example for a new k8ssandra installation:
+
+`helm install k8ssandra k8ssandra/k8ssandra -f backup-restore-values.yaml`
 
-Install the k8ssandra chart with the following properties. Backup and restore operations are enabled by default. In the following example, `bucketName` corresponds to the name of the S3 bucket: `K8ssanda-bucket-dev`.  The `bucketSecret` corresponds to the secret credentials.
+Example for an existing k8ssandra installation:
 
-`helm install k8ssandra-1 k8ssandra/k8ssandra --set backupRestore.medusa.bucketName=k8ssanda-bucket-dev --set backupRestore.medusa.bucketSecret=medusa-bucket-secret`
+`helm upgrade k8ssandra k8ssandra/k8ssandra -f backup-restore-values.yaml`
 
-The `k8ssandra` Helm chart includes cluster services and the Grafana Operator. Notice that `k8ssandra` add a number of properties in the `cassdc` datacenter.  
+Allow a few minutes for the pods to start and proceed to a Ready state; check the pod status periodically:
+
+`kubectl get pods`                           
+
+```
+NAME                                                  READY   STATUS             RESTARTS   AGE
+k8ssandra-cass-operator-66dc79dd97-kvzdq              1/1     Running            44         18h
+k8ssandra-dc1-default-sts-0                           1/2     Running            0          16m
+k8ssandra-grafana-7cb58d4f6b-5dn6g                    2/2     Running            1          18h
+k8ssandra-kube-prometheus-operator-85695ffb-xdq8c     1/1     Running            0          18h
+k8ssandra-medusa-operator-58586cb9c4-qctvb            1/1     Running            5          16m
+k8ssandra-reaper-operator-97754757-p7rz8              1/1     Running            11         18h
+prometheus-k8ssandra-kube-prometheus-prometheus-0     2/2     Running            1          18h
+prometheus-pulsar-kube-prometheus-sta-prometheus-0    2/2     Running            1          18h
+```
+
+Backup and restore operations are enabled by default. In the example YAML, `bucketName` corresponds to the name of the S3 bucket: `K8ssanda-bucket-dev`.  The `bucketSecret` corresponds to the secret credentials.
+
+The `k8ssandra` Helm chart includes the Grafana Operator. Notice that `k8ssandra` adds a number of properties in the `cassdc` datacenter.  
```

Comment:
We no longer include the Grafana operator. We will include Grafana itself though.

Reply:
Edited - thanks